### PR TITLE
"Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Created a new Rails app called 'hello-rails-react'
- Used the command `rails new -j webpack my_new_rails_app` as described
- Created a table for storing 5 different greetings
- Created an API endpoint that selects a random greeting from the table
- Created a static view that is the root of the app
- Created the `App` component with react-router and rendered it in the static view
- Created the `Greeting` component that displays a random greeting
- Created a `store` and a `reducer` that connects to the API endpoint

Screenshot (125)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install Ruby on Rails
+
+### Setup
```
Comment (SajjadAhmad14) [inline README.md:40]: Please add commands to install dependencies, and set up database here as README should have all the information for user to start with the project.

File: app/javascript/src/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Greeting from './components/Greeting';
+
+function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Greeting />} />
```
Comment (SajjadAhmad14) [inline app/javascript/src/App.js:8]: App does not show any greeting but when I try to go to the route `api/v1/greetings` it returns a JSON which is a greeting. Kindly make sure the app shows a random greeting.
tests

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Shahrizoda, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
No linter errors. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/javascript/src/App.js (hunk 2 above)
Comment (shyusu4) [reply app/javascript/src/App.js:8]: Hi @SajjadAhmad14, thank you for the review. I'm able to see the random greeting on my end, so I'm not sure what the issue is. I'll look into it. 👍

**Review by t-yanick — APPROVED** [review]
Hi @shyusu4 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: We documented procedure on how to run your app
:heavy_check_mark:  Good job uploading a screenshot of the greeting displaying at your end in the PR
:heavy_check_mark:  Descriptive PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**